schema.sql

```diff
@@ -2,6 +2,8 @@
 -- Day 1
 CREATE DATABASE vet_clinic;
 
+CREATE DATABASE vet_clinic;
+
```

Comment:
**Changes Required**
We are missing quite a number of requirements. Let us please help add them to the schema.
- [ ] Students should add a new table with the name owners.
- [ ] Students should add a new table with the name `species`.
- [ ] Students should modify the `animals` table